### ISSUE TITLE
Question about SyncBN of Mutil-GPU

### ISSUE THREAD

**Issue by xunhen** [HRNet/HRNet-Object-Detection#10]
Hello!
Did you use syncBN during training this HRNet? 
I find that the mmdetection had not achieved the  function of syncBN in the version you are using.

**Comment by wondervictor** [comment]
Well, our released code does not support SyncBN. All models are trained with fixed batch normalization. If you want to train HRNets with SyncBN, using SyncBN in PyTorch 1.1 might be a great choice. (Just changing all BatchNorm into SyncBatchNorm)

**Comment by xunhen** [comment]
Thank you very much!
